client/src/components/Markdown/Markdown.vue

```diff
@@ -22,15 +22,28 @@
                     Edit
                     <FontAwesomeIcon icon="edit" />
                 </b-button>
+                <b-button
+                    v-b-tooltip.hover
+                    class="float-right markdown-edit mr-2"
+                    role="button"
+                    size="sm"
+                    title="Run Workflow Again"
+                    @click="rerunWorkflow()">
+                    Run Again
+                    <FontAwesomeIcon icon="redo" />
```

Comment:
This isn't a redo as we do for tools

Reply:
So, when the user clicks this button, they should not have to select an input again? They would want it re-run with the input already selected? 

Is that what you mean?

Reply:
> I'd also move this into its own component and then share it between the different sports where we run workflows.

+1 to moving it to it's own component. Are there other places where we already do something like this? Can you give me a pointer? TIA ! :)

Reply:
We don't do workflow re-runs, no. It's a complicated thing to do that we'll want to do, but re-run implies all the same options are chosen, yes. Keep it simple, just running the workflow is fine.